packages/stats-py/aqsha_stats/contract.py

```
@@ -0,0 +1,125 @@
+        "title": title,
+        "columns": list(columns),
+        "rows": [[r3(v) for v in row] for row in rows],
+        "notes": list(notes or []),
+    }
+
+
+def decision(
+    decision_id: str,
+    label: str,
+    rule: str,
+    value: Any,
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)